compareECLFiles.cmake

```diff
@@ -66,22 +60,25 @@ macro (add_test_compareECLInitFiles casename filename simulator)
                            ${COMPARE_SUMMARY_COMMAND}
                            ${COMPARE_ECL_COMMAND}
                TEST_ARGS ${OPM_DATA_ROOT}/${casename}/${filename})
-endmacro (add_test_compareECLInitFiles)
+endmacro (add_test_compare_restarted_simulation)
 
 ###########################################################################
-# TEST: parallelECLFiles
+# TEST: add_test_compare_parallel_simulation
 ###########################################################################
 
 # Input:
 #   - casename: basename (no extension)
 #
-macro (add_test_parallelECLFiles casename filename simulator)
+# Details:
+#   - This test class compares the output from a parallel simulation
+#     to the output from the serial instance of the same model.
+macro (add_test_compare_parallel_simulation casename filename simulator)
   set(abs_tol 0.20)
   set(rel_tol 4e-4)
   set(RESULT_PATH ${BASE_RESULT_PATH}/parallel/${simulator}+${casename})
 
   # Add test that runs flow_mpi and outputs the results to file
-  opm_add_test(parallelECLFiles_${simulator}+${filename} NO_COMPILE
+  opm_add_test(compareECLParallelSim_${simulator}+${filename} NO_COMPILE
```

Comment:
Actually no point in having `ECL` as part of the name here - the simulator is `flow` after all.